knip.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "https://unpkg.com/knip@3/schema.json",
+  "workspaces": {
+    "apps/backend": {
+      "entry": [
+        "src/processes/proc/*.ts",
+        "src/build/bin/*.ts",
+        "src/database/testing/index.ts",
+        "src/graphql/definitions/*.ts"
+      ],
+      "ignore": ["db", "knexfile.js", "src/graphql/__generated__"],
+      "ignoreDependencies": ["pg"],
+      "paths": {
+        "@/*": ["./src/*"]
+      }
+    },
+    "apps/frontend": {
+      "entry": ["src/index.tsx"],
+      "ignore": ["src/gql"],
+      "ignoreDependencies": ["webpack-cli"]
```

Comment:
This should have been detected properly, fixed the bug in Knip v3.5.0 so you don't need to ignore `webpack-cli` if you decide to update.